### ISSUE TITLE
Not working? 

### ISSUE THREAD

**Issue by faqu1** [UnamSanctam/UnamDownloader#27]
I tried to download the miner through this program, nothing happens, everything worked before with the same settings

**Comment by UnamSanctam** [comment]
Is your antivirus removing/blocking it? Does the file get downloaded at all?

**Comment by faqu1** [comment]
But before I loaded the same build, all the same settings, everything worked fine

**Comment by UnamSanctam** [comment]
Is the URL correct? Administrator rights won't stop it from being downloader.

**Comment by faqu1** [comment]
yeah, url correct. The file is on bitbucket. url - ![image](https://user-images.githubusercontent.com/95179319/160629859-7ffe9517-b8d7-424c-b5f7-3fb331c29173.png)

**Comment by UnamSanctam** [comment]
And if you open that URL in an incognito window it instantly downloads the file (you don't have to do anything except maybe select the place you want to save it)?

**Comment by faqu1** [comment]
that was the problem. bitbucket required registration. Thanks for the solution)

**Comment by faqu1** [comment]
which Obfuscator you can advise for unamdownloader?
